katas/content/qubit/index.md

```diff
@@ -106,15 +106,15 @@ Dirac notation is a shorthand notation that eases writing quantum states and com
     </tr>
 </table>
 
-These two kets represent basis states, so they can be used to represent any other state:
+The kets $\ket{0}$ and $\ket{1}$ represent basis states, so they can be used to represent any other state:
 
 $$\begin{bmatrix} \alpha \\ \beta \end{bmatrix} = \alpha\ket{0} + \beta\ket{1}$$
 
-Dirac notation is not only restricted to vectors $0$ and $1$; it can be used to represent any vector, similar to how variable names are used in algebra. For example, we can call the state above "the state $\psi$" and write it as:
+Dirac notation isn't restricted to vectors $0$ and $1$; it can be used to represent any vector, similar to how variable names are used in algebra. For example, you can call the above state "$\psi$" and write it as:
 
 $$\ket{\psi} = \alpha\ket{0} + \beta\ket{1}$$
 
-Several ket symbols have a generally accepted use, so you will see them often:
+Several ket symbols have a generally accepted use, so you will see them often. For example, the following kets are used to represent the states that are superpositions of the basis states:
```

Comment:
This addition looks odd, since most kets are used to represent the states that are superpositions of the basis states... How about
```suggestion
Several ket symbols have a generally accepted use, so you will see them often. For example, the following kets are commonly used:
```